solr/core/src/java/org/apache/solr/handler/component/SearchHandler.java

```diff
@@ -289,6 +295,19 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
       rb.requestInfo.setResponseBuilder(rb);
     }
 
+    //TODO: Should this be for indexing requests as well?
```

Comment:
I think we would want to have two separate controls for this.

Reply:
++